apps/sms/test/marionette/lib/views/inbox/view.js

```diff
@@ -36,9 +36,11 @@ InboxView.prototype = {
   goToConversation: function(conversationId) {
     var conversation = this.accessors.findConversation(conversationId);
 
+    var isConversationLessDraft = !!conversation.getAttribute('data-draft-id');
```

Comment:
We change document with the tap, so if we don't get attribute value before we tap - we'll get StaleElementException :/